tests/unit/amo/components/TestSearchFilters.js

```diff
@@ -244,4 +250,20 @@ describe(__filename, () => {
 
     expect(root.find('.SearchFilters-AddonType')).toHaveLength(0);
   });
+
+  it('sets themes filters shelf with the ADDON_TYPE_THEMES_FILTER filter if static theme is enabled', () => {
+    const fakeConfig = getFakeConfig({ enableStaticThemes: true });
+    const root = render({ _config: fakeConfig });
+    expect(root.instance().addonTypeOptions()[3].value).toEqual(
```

Comment:
This test is brittle because it relies on `addonTypeOptions()` which is just an implementation detail of the component. Instead, it should look at the `<Select \>` and `<option />` components in the rendered output using standard enzyme selectors. You may need a map function to find the value you want but it should be possible.

Reply:
I see... ok let me know how i have it now  works.